### PR TITLE
Add tooltips indicating that swaps are disabled on custom networks

### PR DESCRIPTION
**Pull request by 0xDaedalus**
Closes https://github.com/tahowallet/extension/issues/3191

https://user-images.githubusercontent.com/94649004/228350805-14f99e70-f788-4dbf-ba53-08870ed4faca.mov

### To Test
- [x] Add a custom network with a balance on it.
- [x] Check to make sure a swaps disabled tooltip shows up in the 3 places discussed in #3191

## Testing Env

```
SUPPORT_CUSTOM_NETWORKS=true
SUPPORT_CUSTOM_RPCS=true
```

Latest build: [extension-builds-3205](https://github.com/tahowallet/extension/suites/12366561860/artifacts/656810172) (as of Thu, 20 Apr 2023 11:15:51 GMT).

### CODE REVIEW

File: ui/components/Shared/SharedIconRouterLink.tsx [hunk 1]
```diff
@@ -5,10 +5,27 @@ type Props = {
...
+      // @TODO Make accessible
+      // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
+      <div
+        onClick={(e) => {
+          e.preventDefault()
+          e.stopPropagation()
+        }}
+        className="icon_wrapper"
+      >
+        <i className={`asset_icon ${iconClass}`} />
```
Comment (hyphenized) [inline ui/components/Shared/SharedIconRouterLink.tsx:25]: This icon still changes color on hover

**Review by hyphenized — DISMISSED** [review]
Looks good, two n/b issues:
- The swap button in the asset screen looks shorter when disabled, this might be due to the tooltip affecting layout
- The asset list item action still changes color when hovered, there are global styles on a parent component causing this

**Review by jagodarybacka — COMMENTED** [review]
image
Well Rootstock is not a custom network, also this tooltip is a little bit too large IMO, maybe it would be nice to change copy or font size? 

Also spacing between icons is different when swaps are disabled

**Comment by 0xDaedalus** [comment]
Will leave it up to @VladUXUI to comment on the copy, fixed the font size but was unable to fix the jitter issue.

**Comment by VladUXUI** [comment]
> Will leave it up to @VladUXUI to comment on the copy, fixed the font size but was unable to fix the jitter issue.

It's a good point. Let's go with "swaps aren't supported on this network" as it covers all networks that don't support swaps

**Comment by kkosiorowska** [comment]
I'll take a look at it again 👀

**Review by kkosiorowska — APPROVED** [review]
The copy has been updated. Let's not block QA. Works as expected. ✅

The following issues will be resolved in separate PRs.

> The swap button in the asset screen looks shorter when disabled, this might be due to the tooltip affecting layout

> Also spacing between icons is different when swaps are disabled

**Pull request merged**